src/wled2.html

```diff
@@ -186,70 +228,86 @@
+        </label>
         <input type="color" id="node-input-color3">
       </div>
       <div class="form-row">
-        <label for="node-input-effect"><i class="icon-server"></i> Effect</label>
+        <label for="node-input-effect">
+          <input style="margin: 0; width: auto;" type="checkbox" title="Enable?" alt="Enable?" id="node-input-enableEffect" />
+          <i class="icon-server"></i> Effect
+        </label>
         <select id="node-input-effect"></select>
     </div>
     <div class="form-row">
-      <label for="node-input-effectSpeed"><i class="icon-tag"></i> Effect speed</label>
+      <label for="node-input-effectSpeed">
+        <input style="margin: 0; width: auto;" type="checkbox" title="Enable?" alt="Enable?" id="node-input-enableEffectSpeed" />
+        <i class="icon-tag"></i> Effect speed
+      </label>
       <input type="range" id="node-input-effectSpeed" min="0" max="255">
     </div>
     <div class="form-row">
-      <label for="node-input-effectIntensity"><i class="icon-tag"></i> Effect intensity</label>
+      <label for="node-input-effectIntensity">
+        <input style="margin: 0; width: auto;" type="checkbox" title="Enable?" alt="Enable?" id="node-input-enableEffectIntensity" />
+        <i class="icon-tag"></i> Effect intensity
+      </label>
       <input type="range" id="node-input-effectIntensity" min="0" max="255">
     </div>
 
       <div class="form-row">
-        <label for="node-input-palette"><i class="icon-server"></i> Palette</label>
+        <label for="node-input-palette">
+          <input style="margin: 0; width: auto;" type="checkbox" title="Enable?" alt="Enable?" id="node-input-enablePalette" />
+          <i class="icon-server"></i> Palette
+        </label>
         <select id="node-input-palette"></select>
     </div>
     <div class="form-row">
-      <label for="node-input-preset"><i class="icon-tag"></i> Preset</label>
-      <select id="node-input-preset">
-        <option value="">None</option>
-        <option value="1">1</option>
-        <option value="2">2</option>
-        <option value="3">3</option>
-        <option value="4">4</option>
-        <option value="5">5</option>
-        <option value="6">6</option>
-        <option value="7">7</option>
-        <option value="8">8</option>
-        <option value="9">9</option>
-        <option value="10">10</option>
-        <option value="11">11</option>
-        <option value="12">12</option>
-        <option value="13">13</option>
-        <option value="14">14</option>
-        <option value="15">15</option>
-        <option value="16">16</option>
-      </select>
+      <label for="node-input-preset">
+        <input style="margin: 0; width: auto;" type="checkbox" title="Enable?" alt="Enable?" id="node-input-enablePreset" />
+        <i class="icon-tag"></i> Preset
+      </label>
+      <input type="number" step="1" min="1" max="65535" placeholder="0" id="node-input-preset" />

```

Comment:
Max should match the max value supported by WLED.

Reply:
This is now the max supported by WLED as of 0.11.0 :)